### PR TITLE
Added health endpoint

### PR DESCRIPTION
**Pull request by nishanthsanjeev**
Added health endpoint to check the health of the Kubernetes cluster.

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -13,6 +13,15 @@
 from . import app
 
 
+############################################################
+# Health Endpoint
+############################################################
+@app.route("/health")
```
Comment (rachitjain2706) [inline service/routes.py:19]: Can you add `methods=["GET"]` to this?

File: service/routes.py [hunk 2]
```diff
@@ -13,6 +13,15 @@
...
+def health():
+    """Health Status"""
```
Comment (rachitjain2706) [inline service/routes.py:21]: How about adding it to the logger as well?

File: service/routes.py (hunk 1 above)
Comment (nishanthsanjeev) [reply service/routes.py:19]: Done!

File: service/routes.py (hunk 2 above)
Comment (nishanthsanjeev) [reply service/routes.py:21]: Changes made!

**Review by sahildeshpande25 — APPROVED** [review]
Build successful and endpoint works as expected

**Pull request merged**